docs/faq/general/how-do-i-allowlist-codacy-on-my-git-provider.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Enable static IP addresses for your Codacy Cloud organization so that you can allowlist Codacy on your Git provider.
+---
+
+# How do I allowlist Codacy on my Git provider?
+
+{%
+    include-markdown "../../assets/includes/paid.md"
+    start="<!--start-paid-feature-->"
+    end="<!--end-paid-feature-->"
+%}
+
+If you require an additional layer of security and control on your Git provider, you can configure an allowlist containing the specific IP addresses that are able to access your Git repositories and resources.
+
+To allowlist Codacy Cloud on your Git provider:
+
+1.  Send an email to <span class="skip-vale">[support@codacy.com](mailto:support@codacy.com?subject=Enabling static IP addresses)</span> asking us to enable static IP addresses for your organization.
+
+1.  After receiving a confirmation that static IP addresses are active for your Codacy Cloud organization, add the following IP addresses to the allowlist on your Git provider:
+
+    -   0.0.0.0
+    -   0.0.0.0
```

Comment:
I was about to ask that. Not really sure if we should reveal these IP addresses or just mention something like "upon confirmation we'll provide you the IP addresses for the allowlist"

Reply:
If possible, it will be simpler to just list the static IPs publicly (customers can easily obtain them, and we don't have to remember/do an extra step to include them in the confirmation reply). But if there could be any disadvantage such as security concerns, we can ask customers to wait for our reply.

I would like to know what @medleycodacy or @h314to think about this. :+1:

Reply:
those are aws nat gateway addresses, so I don't think there is any issue with putting them on the documentation